### DISCUSSION TITLE
Can't embed 'generated' image

### DISCUSSION THREAD

**Discussion by kkparkclouflake** [outline/outline#2375] [Self Hosting]
Sort of site like 'kroki.io' generates Image URL like this;
https://kroki.io/plantuml/svg/eNplj0FvwjAMhe_5FVZP40CgaNMuUGkcdttp3Kc0NSVq4lRxGNKm_fe1HULuuD37-bOfuXPUm2QChEjRnlIMCDmdUfHNSYY6xh42a9Fsegflk-yYlOLlcHK2I2SGtX4WZm9sZ1o8uOzxxbuWAlIGj8cshs6M1jDuY2owyU2P8jAezdnn10j53X0hlBsZFW021Pq7HaVSNw-KN-OogG8F8BAGqT8dXhZjxW4cyJEW6kcC-yHWFagHqW0MfaThhYmaVyE26P_x27qaDmXeruqqAMMw1h-ZlRI4aF3dX7hOwm5XzfIKDctlNcshPT1tFa8JPYAj-Zf5F065sqM=

That URL gives SVG image, so It must can embed to document.
But Markdown image embed tag won't work with that URL.
Maybe problem caused by URL because It's not ended with extension... I don't know.

I can embed images using Typora. Embed image to Typora, and Copy&Paste are work.
There's exist other easy workaround to embed images?

```
![Image](https://kroki.io/plantuml/svg/eNplj0FvwjAMhe_5FVZP40CgaNMuUGkcdttp3Kc0NSVq4lRxGNKm_fe1HULuuD37-bOfuXPUm2QChEjRnlIMCDmdUfHNSYY6xh42a9Fsegflk-yYlOLlcHK2I2SGtX4WZm9sZ1o8uOzxxbuWAlIGj8cshs6M1jDuY2owyU2P8jAezdnn10j53X0hlBsZFW021Pq7HaVSNw-KN-OogG8F8BAGqT8dXhZjxW4cyJEW6kcC-yHWFagHqW0MfaThhYmaVyE26P_x27qaDmXeruqqAMMw1h-ZlRI4aF3dX7hOwm5XzfIKDctlNcshPT1tFa8JPYAj-Zf5F065sqM=)
```

**Comment by tommoor** [comment]
Hey, I'm not really sure why it's not working for you – I'm able to copy and paste that markdown into a document without problem and the image loads :) 

Perhaps you can find the error in the developer console?

**Reply by kkparkclouflake** [reply]
Hmm... I resolve WebSocket errors but It's still not working...

Firstly, I'm using Latest Firefox, With Manjaro Linux, and Serve my own Outline on Docker with NGINX.

No console errors, Copy & Paste works good (Works any text editors)
But didn't work with manually typed Markdown.

Firstly, Write
`![Image](`
and Paste Image URL, than write closing
`)`
It doesn't working.

I can't test insert Image link after done `![Image]()` writing, because It directly converted to Image Embed. 

Maybe it occurred by Hyperlink converting. The link converted to Hyperlink when It pasted.
I attach the screenshot.

![image](https://user-images.githubusercontent.com/87506526/127252109-44273cef-d02b-41d6-aaf4-b1fa49e544e4.png)

**Reply by whuwangyong** [reply]
> I can't test insert Image link after done `![Image]()` writing, because It directly converted to Image Embed.

I'm also troubled by this problem
